sos/plugins/cs.py

```diff
@@ -1,128 +1,140 @@
-# Copyright (C) 2007-2010 Red Hat, Inc., Kent Lamb <klamb@redhat.com>
-#                                        Marc Sauton <msauton@redhat.com>
-#                                        Pierre Carrier <pcarrier@redhat.com>
-
-# This program is free software; you can redistribute it and/or modify
-# it under the terms of the GNU General Public License as published by
-# the Free Software Foundation; either version 2 of the License, or
-# (at your option) any later version.
-
-# This program is distributed in the hope that it will be useful,
-# but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-# GNU General Public License for more details.
-
-# You should have received a copy of the GNU General Public License along
-# with this program; if not, write to the Free Software Foundation, Inc.,
-# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+# Copyright (C) 2018 Brian Gribble <bgribble@redhat.com>, Red Hat.
+# All rights reserved.
+#
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions
+# of the version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further
+# information.
 
 from sos.plugins import Plugin, RedHatPlugin
 from os.path import exists
 from glob import glob
+import os
 
 
 class CertificateSystem(Plugin, RedHatPlugin):
-    """Certificate System and Dogtag
+    """ Certificate System and Dogtag
     """
 
     plugin_name = 'cs'
     profiles = ('identity', 'security')
 
     packages = (
-        "redhat-cs",
-        "rhpki-common",
-        "pki-common",
-        "redhat-pki",
-        "dogtag-pki",
-        "pki-base"
+        'redhat-cs',
+        'rhpki-common',
+        'pki-common',
+        'pki-base'
     )
 
     files = (
-        "/opt/redhat-cs",
-        "/usr/share/java/rhpki",
-        "/usr/share/java/pki"
+        '/opt/redhat-cs',
+        '/usr/share/java/rhpki',
+        '/usr/share/java/pki'
     )
 
     def checkversion(self):
-        if self.is_installed("redhat-cs") or exists("/opt/redhat-cs"):
+        """ Check if Certificate System 7, 8, or 9 is installed.
+        """
+
+        if self.is_installed('redhat-cs') or exists('/opt/redhat-cs'):
             return 71
-        elif self.is_installed("rhpki-common") or \
-                len(glob("/var/lib/rhpki-*")):
+        elif self.is_installed('rhpki-common') or \
+                len(glob('/var/lib/rhpki-*')):
```

Comment:
Is there really any need to change all this quoting? It makes all your other changes much harder to spot and adds needless churn to this patch: the correct way to make this kind of change is to do it in a separate patch, either as cleanup before the new change, or to make things consistent afterwards. That allows reviewers to focus on the actual content change rather than line after line of quote twiddling.